metrics/sinks/influxdb/influxdb.go

```diff
@@ -198,16 +198,16 @@ func (sink *influxdbSink) createDatabase() error {
 	if sink.dbExists {
 		return nil
 	}
+
 	q := influxdb.Query{
-		Command: fmt.Sprintf(`CREATE DATABASE %s WITH NAME "default"`, sink.c.DbName),
+		Command: fmt.Sprintf(`CREATE DATABASE %s WITH DURATION %s REPLICATION 1 NAME "default"`, sink.c.DbName, sink.c.RetentionPolicy),
 	}
 
-	if resp, err := sink.client.Query(q); err != nil {
-		if !(resp != nil && resp.Err != nil && strings.Contains(resp.Err.Error(), "already exists")) {
-			err := sink.createRetentionPolicy()
-			if err != nil {
-				return err
-			}
+	if resp, err := sink.client.Query(q); err != nil || resp.Error() != nil {
+		if err != nil {
```

Comment:
From https://docs.influxdata.com/influxdb/v1.2/query_language/database_management/#create-database 
"A successful CREATE DATABASE query returns an empty result. If you attempt to create a database that already exists, InfluxDB does nothing and does not return an error."

However this message is not present in doc of influxdb v0.9 and I've tested it with v1.2.
I can do additional tests with older version of influxdb.
What version of influxdb do we support?